bucket/multibootusb.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "9.2.0",
+    "description": "Create multiboot live Linux on a USB disk.",
+    "homepage": "https://github.com/mbusb/multibootusb",
+    "license": "GPL-2.0-or-later",
+    "url": "https://github.com/mbusb/multibootusb/releases/download/v9.2.0/multibootusb-9.2.0-setup.exe#/dl.7z",
+    "hash": "0e9108b4f95562727d0e557e86e7b8f4b83cf942303bd67c2350d8820289fe17",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\multibootusb.log\")) { New-Item \"$dir\\multibootusb.log\" | Out-Null }",
+        "Remove-Item  \"$dir\\uninst.exe\""
+    ],
+    "shortcuts": [
+        [
+            "multibootusb.exe",
+            "MultiBootUSB"
+        ]
+    ],
+    "persist": "multibootusb.log",
+    "checkver": "github",
```

Comment:
Should we remove checkver as the project seems discontinued (last updated 2019) and the website is down? See: https://github.com/mbusb/multibootusb/issues/536

Reply:
> Should we remove checkver as the project seems discontinued (last updated 2019)

Agree.